src/net/sourceforge/kolmafia/request/CharSheetRequest.java

```diff
@@ -702,31 +702,46 @@ public static List<ParsedSkillInfo> parseSkills(final String responseText, boole
     }
   }
 
-  public static void parseStatus(final JSONObject JSON) throws JSONException {
-    int muscle = JSON.getIntValue("muscle");
-    int mysticality = JSON.getIntValue("mysticality");
-    int moxie = JSON.getIntValue("moxie");
+  public static void parseStatus(final JSONObject json) throws JSONException {
+    int muscle = json.getIntValue("muscle");
+    int mysticality = json.getIntValue("mysticality");
+    int moxie = json.getIntValue("moxie");
     long rawmuscle;
     long rawmysticality;
     long rawmoxie;
     if (KoLCharacter.inGreyYou() || KoLCharacter.inZootomist()) {
       // Raw values are more precise, but they don't exist in Grey You and are wrong in Zooto
-      long basemuscle = JSON.getLong("basemuscle");
+      long basemuscle = json.getLong("basemuscle");
       rawmuscle = basemuscle * basemuscle;
 
-      long basemysticality = JSON.getLong("basemysticality");
+      long basemysticality = json.getLong("basemysticality");
       rawmysticality = basemysticality * basemysticality;
 
-      long basemoxie = JSON.getLong("basemoxie");
+      long basemoxie = json.getLong("basemoxie");
       rawmoxie = basemoxie * basemoxie;
+
+      var grafts = json.getJSONObject("grafts");
+      if (KoLCharacter.inZootomist() && grafts != null) {
```

Comment:
CharSheetRequest is maybe not the best place for this, but no harm I think.

Reply:
actually yeah this should go in `ApiRequest.parseStatus` directly, because it doesn't do anything for the character sheet.

Reply:
Fine fine